app/src/main/java/io/token/sample/Application.java

```diff
@@ -119,60 +112,61 @@ public static void main(String[] args) throws IOException {
             Map<String, String> formData = gson.fromJson(req.body(), type);
             String callbackUrl = req.scheme() + "://" + req.host() + "/redeem-standing-order-popup";
 
-            String tokenRequestUrl = initializeStandingOrderTokenRequestUrl(
-                    formData,
-                    callbackUrl,
-                    res);
+            String tokenRequestUrl =
```

Comment:
@priyanshu-ap can you make sure we don't change the style please. This just adds lot more changes than needed to be reviewed.